.github/actions/git-commit-changes/action.yml

```diff
@@ -17,19 +17,40 @@ inputs:
     description: "Additional Git commit command options"
     required: false
     default: ''
+  ignore-empty:
+    description: "Ignore empty commits"
+    required: false
+    default: 'false'
 runs:
   using: composite
   steps:
-    - name: Commit stagged changes
+    - name: Configure git user
+      shell: bash
+      run: |
+        git config --global user.name ${{ inputs.username }}
+        git config --global user.email ${{ inputs.username }}@users.noreply.github.com
```

Comment:
there is already an action for this
https://github.com/Alfresco/alfresco-build-tools#configure-git-author

in general it's better to not embed an action inside another action, just use the existing action before this action in your workflow

Reply:
Yes, this generally true, but in some cases it makes sense to group together several actions that are always called together, specially when they are called in several places.
Note that the current action is already like that, these two lines were just moved up. Given the fact that this action is used in a lot of places, it's not viable to split them in the callers now. However, maybe the two lines can be replaced by a call to `configure-git-author`.